content/hardware/05.pro-solutions/solutions-and-kits/edge-control/features.md

```diff
@@ -15,13 +15,43 @@ The **Edge control** can be positioned anywhere and is suitable for precision fa
 
 <Feature title="MKR slots" image="mkr-form-factor">
 
-  The on-board MKR slots 1 & 2 can be used to connect Arduino MKR boards to extend the capabilities such as connectivity through LoRa, Wi-Fi, 2G/3G/CatM1/NBIoT, and Sigfox.
+  The on-board MKR slots 1 & 2 can be used to connect Arduino MKR boards to extend the capabilities such as connectivity through LoRa®, Wi-Fi®, 2G/3G/CatM1/NBIoT, and Sigfox.
 
 </Feature>
 
-<Feature title="Smart agriculture" image="humidity-sensor">
+<Feature title="Watermark Sensors" image="humidity-sensor">
 
-  Particularly suited to smart agriculture, the sensors can collect real-time data such as weather conditions, soil quality, crop growth, amongst others. Once sent to the Arduino Cloud, the data value chain becomes valuable analytics that supports business processes at various levels (e.g. crop yield, equipment efficiency, staff performance, etc.). The Arduino Edge Control has the capability to improve crop quality and reduce human effort/error by automating processes like irrigation, fertilization, or pest control.
+  The Edge Control can handle up to 16 watermark sensor inputs, being a suitable solution for soil humidity measurements on big crop fields.
+
+</Feature>
+
+<Feature title="Solid State Relays" image="settings">
+
+  The Edge Control features four galvanically isolated solid state relays (SSR) that can handle up to 2.5A and 60V AC. 
+
+</Feature>
+
+<Feature title="4-20mA Sensors" image="communication">
+
+  The Edge Control has four 4-20mA sensor inputs, perfect to support a vast variety of industrial sensors. 
```

Comment:
Try to solve this format problem
![image](https://github.com/arduino/docs-content/assets/7687624/8bdc780c-22a4-4a50-b224-812fe3a890d6)
where 4-20 mA is separed